integration/bin/tachyon-yarn-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# prepare Tachyon dir structure for a YARN container to launch master or worker
+tar zxf tachyon.tar.gz
+
+# launch master or worker on this container
+if [ $1 = 'master' ]; then
```

Comment:
`[[ "$1" == "master" ]]` for consistency with shell style used in the rest of our codebase